Src/Services/RoutesService.php

```diff
@@ -20,6 +20,7 @@ public function frontEndRoutes($plugin)
 
         Route::group(['CheckWebhookSecret'], function () {
             Route::post('invoice-create', 'Frontend\InvoicesController@create');
+            Route::post('webhook-create', 'Frontend\WebhookController@create');
```

Comment:
[blocking]

We need to validate x-mondu-signature for this route

Please take a look how it's implemented in other plugins.

https://github.com/mondu-ai/checkout-shopware5/blob/8fbab39649b8a95de3d3e4890d045512bc2f0e96/Controllers/Frontend/MonduWebhook.php#L95

Reply:
Done